### PR TITLE
front: add upgrade STDCM div

### LINKED ISSUE #8127: STDCM new interface : Add "Upgrade STDCM" div
The buttons must be clickable, but nothing must append. For now, it's all about display.

Mock-up : 

Image

### PR DESCRIPTION
**Pull request by theocrsb**
close #8127

### CODE REVIEW

File: front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx
Comment (Uriel-Sautron) [inline front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx:1]: The comment-16 icon is missing after the title

File: front/src/styles/scss/applications/stdcmV2/_home.scss [hunk 1]
```diff
@@ -263,6 +266,38 @@
         }
       }
     }
+    .upgrade {
+      display: flex;
+      flex-direction: column;
+      margin-left: 2rem;
+      margin-right: 2rem;
```
Comment (Uriel-Sautron) [inline front/src/styles/scss/applications/stdcmV2/_home.scss]:
```suggestion
      margin: 0 2rem;
```

**Review by Uriel-Sautron — COMMENTED** [review]
Nice work just a few comments

File: front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const StdcmUpgrade = () => {
+  const { t } = useTranslation('stdcm');
+  return (
+    <div className="upgrade">
+      <div className="text">
+        <div className="title">{t('simulation.results.upgrade.helpUs')}</div>
+        <div>{t('simulation.results.upgrade.reason')}</div>
+      </div>
+      <div className="buttons-list">
+        <button
```
Comment (kmer2016) [inline front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx]: you can use `Button` from @osrd-project/ui-core`

File: front/src/styles/scss/applications/stdcmV2/_home.scss (hunk 1 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/stdcmV2/_home.scss]: Done

File: front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx
Comment (theocrsb) [reply front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx:1]: Done

File: front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx (hunk 2 above)
Comment (theocrsb) [reply front/src/applications/stdcmV2/components/StdcmResults/StdcmUpgrade.tsx]: Sure, done

**Review by kmer2016 — APPROVED** [review]
LGTM and tested !

File: front/src/styles/scss/applications/stdcmV2/_home.scss [hunk 3]
```diff
@@ -263,6 +266,44 @@
...
+      .text {
+       color: rgb(0, 0, 0);
+       margin-left: 0.5rem;
+       .title{
+       font-weight: 600;
+       font-size: 1.125rem;
+       padding-top: 1.563rem;
+       display: flex;
+       flex-direction: row;
+        .comment-icon {
```
Comment (Uriel-Sautron) [inline front/src/styles/scss/applications/stdcmV2/_home.scss]: Add a display: flex, to get the same rendering as on sketch

Comment (theocrsb) [reply front/src/styles/scss/applications/stdcmV2/_home.scss]: Done

File: front/src/styles/scss/applications/stdcmV2/_home.scss [hunk 4]
```diff
@@ -179,7 +179,10 @@
       .table-container {
         margin-top: 2.125rem;
         margin-left: 2rem;
+        margin-right: 2rem;
```
Comment (Uriel-Sautron) [inline front/src/styles/scss/applications/stdcmV2/_home.scss]:
```suggestion
        margin: 0 2rem;
```

Comment (theocrsb) [reply front/src/styles/scss/applications/stdcmV2/_home.scss]: Done

**Review by Uriel-Sautron — APPROVED** [review]
LGTM !!

**Pull request merged**